kfs-core/src/main/java/edu/arizona/kfs/gl/batch/BudgetAdjustmentMergeSortStep.java

```diff
@@ -0,0 +1,140 @@
+package edu.arizona.kfs.gl.batch;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Date;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.SuffixFileFilter;
+import org.kuali.kfs.gl.batch.BatchSortUtil;
+import org.kuali.kfs.gl.batch.ScrubberSortComparator;
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.arizona.kfs.gl.GeneralLedgerConstants;
+
+public class BudgetAdjustmentMergeSortStep extends AbstractStep {
+	private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(BudgetAdjustmentMergeSortStep.class);
+	
+	protected String batchFileDirectoryName;
+	
+	@Override
+	public boolean execute(String jobName, Date jobRunDate) {
```

Comment:
I agree. Class IcrEncumbranceSortStep is an example of LOG.debug and LOG.info being used in a batch step class.